### PR TITLE
Adding separate phased learning rate options and defaults

### PR DESCRIPTION
**Pull request by Maxusmusti**
Added 2e-5 as phase1 knowledge default and 6e-6 as phase2 skills default

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the
  [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary).
- [ ] [Changelog](https://github.com/instructlab/instructlab/blob/main/CHANGELOG.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Functional tests have been added, if necessary.
- [ ] E2E Workflow tests have been added, if necessary.

### CODE REVIEW

File: src/instructlab/cli/model/train.py [hunk 1]
```diff
@@ -397,10 +397,12 @@ def train(
     phased_phase1_data: pathlib.Path | None,
     phased_phase1_num_epochs: int | None,
     phased_phase1_samples_per_save: int | None,
+    phased_phase1_learning_rate: float | None,
```
Comment (jaideepr97) [inline src/instructlab/cli/model/train.py:400]: might need to add a click flag for this as well that should read the default from the config file

File: src/instructlab/cli/model/train.py [hunk 2]
```diff
@@ -397,10 +397,12 @@ def train(
...
     phased_phase1_effective_batch_size: int | None,
     phased_phase2_data: pathlib.Path | None,
     phased_phase2_num_epochs: int | None,
     phased_phase2_samples_per_save: int | None,
+    phased_phase2_learning_rate: float | None,
```
Comment (jaideepr97) [inline src/instructlab/cli/model/train.py:405]: same

File: src/instructlab/cli/model/train.py (hunk 1 above)
Comment (Maxusmusti) [reply src/instructlab/cli/model/train.py:400]: adding now

File: src/instructlab/cli/model/train.py (hunk 2 above)
Comment (Maxusmusti) [reply src/instructlab/cli/model/train.py:405]: adding now

File: src/instructlab/cli/model/train.py (hunk 1 above)
Comment (Maxusmusti) [reply src/instructlab/cli/model/train.py:410]: added

File: src/instructlab/cli/model/train.py (hunk 2 above)
Comment (Maxusmusti) [reply src/instructlab/cli/model/train.py:415]: added

**Comment by Maxusmusti** [comment]
Updating for ruff formatting now

**Comment by Maxusmusti** [comment]
Ok formatting / linting all good

**Review by JamesKunstle — APPROVED** [review]
nice, pass-through looks right to me

**Review by jaideepr97 — APPROVED** [review]

File: src/instructlab/configuration.py [hunk 3]
```diff
@@ -507,6 +507,11 @@ class _train(BaseModel):
         ge=0,
         description="Number of samples the model should see before saving a checkpoint during phase1. Disabled when set to 0.",
     )
+    phased_phase1_learning_rate: float = Field(
+        default=2e-5,
+        ge=0,
```
Comment (cdoern) [inline src/instructlab/configuration.py:512]: what is ge? I have never seen this before in a flag

**Review by cdoern — COMMENTED** [review]
one question

File: src/instructlab/configuration.py (hunk 3 above)
Comment (Maxusmusti) [reply src/instructlab/configuration.py:512]: greater than or equal to

Comment (Maxusmusti) [reply src/instructlab/configuration.py:512]: and gt is just greater than
